.tina/config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'tinacms';
+import { schema } from './schema';
+const branch = process.env.NEXT_PUBLIC_TINA_BRANCH || process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF || process.env.HEAD || '';
+
+export default defineConfig({
+  build: {
+    outputFolder: 'admin',
+    publicFolder: 'public',
+  },
+  schema,
+  branch,
```

Comment:
is it still possible to use tina locally? I don't want to push user to use tina cloud

Reply:
@bmstefanski What is the error you are seeing?

Reply:
Everything should work locally

Reply:
Looking into this more, are you talking about building the App without needing Tina Cloud integration? 

Right now we promote using Tina Cloud on the server side (even on CI). I am guessing that vercel error is something like this

```
Error: Client not configured properly. Missing branch, clientId, token. Please visit https://tina.io/docs/tina-cloud/connecting-site/ for more information
```    
This is because to build your app you need to query tina cloud. 


If you want to build the app without Tina Cloud the build command would have to look like this.

```
build: "tinacms dev -c 'next build' "
```

However this would not build admin interface for production.  


I could make this the default build script (so the site will build without TinaCloud) but update the Readme to add docs of how to edit the script so the site can be used with TinaCloud and you can edit in production.

Reply:
I got this error when deploying to vercel or trying to build in local
```
Error occurred prerendering page "/admin/[[...tina]]". Read more: https://nextjs.org/docs/messages/prerender-error
Error: Unable to find `TinaProvider`; did you forget to add the TinaCMS container to your app root?  See our setup docs: https://tina.io/docs/introduction/tina-init/#adding-tina
```